keystone_spassword/contrib/spassword/backends/sql.py

```diff
@@ -27,8 +27,8 @@
 from keystone_spassword.contrib.spassword import Driver
```

Comment:
Does this PR needs of changelog entry?

Reply:
fixed in 3eb8d9f9dd9eef8bf8f5dec0ba828c389b80ac7b